errbot/backends/irc.py

```diff
@@ -343,6 +346,12 @@ def on_pubmsg(self, _, e):
         msg.nick = msg.frm.nick  # FIXME find the real nick in the channel
         self.callback.callback_message(msg)
 
+        possible_mentions = re.findall(IRC_NICK_REGEX, e.arguments[0])
+        room_users = self.channels[room].users()
+        mentions = filter(lambda x: x in room_users, possible_mentions)
+        if mentions:
+            self.callback.callback_mention(msg, mentions)
```

Comment:
self.callback.callback_mention(msg, [self.build_identifier(mention) for mention in mentions]) to pass IRCIdentifiers to the plugins instead of strings no ?